ocaml/Makefile.common-jst

```diff
@@ -172,6 +173,12 @@ _install: compiler
 install: _install
 	mkdir -p '$(prefix)'
 	rsync --chmod=u+rw,go+r -rl _install/ '$(prefix)'
+	rm '$(prefix)/lib/ocaml/ocaml_compiler_internal_params'
```

Comment:
This is all unfortunate (as we've discussed). I've put it on the agenda for this week's group meeting, to see if we can find a better way. And I've made an internal ticket.